src/gazebo_camera_manager_plugin.cpp

```diff
@@ -322,6 +346,9 @@ void CameraManagerPlugin::_handle_message(mavlink_message_t *msg, struct sockadd
             case MAV_CMD_REQUEST_STORAGE_INFORMATION:
                 _handle_storage_info(msg, srcaddr);
                 break;
+            case MAV_CMD_REQUEST_MESSAGE:
```

Comment:
are the messages here camera control messages, such as take picture, shoot video?   If yes, may we make it a bit more specific, such as MAV_CMD_REQUEST_CAMERA_CONTROLS or MAV_CMD_REQUEST_CONTROL_MESSAGE, also give _handle_request_message a bit specific name.

Reply:
Not quite. One of MavLink's command message types is just called ["Request Message"](https://mavlink.io/en/messages/common.html#MAV_CMD_REQUEST_MESSAGE). This type of command will include a [message ID](https://mavlink.io/en/messages/common.html#messages), and essentially asks the camera to respond to the command with a message type matching that message ID. The "MAV_CMD_REQUEST_MESSAGE" case is simply there to handle a Request Message command. In the _handle_request_message() function, I have another switch block that narrows down which Message ID is being requested. This could be a wide variety of messages, but I only care about requests for capture information.